### DISCUSSION TITLE
data-testid is not passed down after wrapping with controller

### DISCUSSION THREAD

**Discussion by wangssie** [react-hook-form/react-hook-form#5018] [Q&A]
Resharing the issue here: https://spectrum.chat/react-hook-form/help/data-testid-is-not-recognised-in-test-file-after-wrapping-with-controller~a5511a7d-e6e3-4c70-8ffe-67f0014cf205

I'm unable to pass down the data-testid to the child components wrapped by <Controller/>

Is there a way to do this? Or potentially an alternative method for accessing the component so I can use it for testing? 

Thanks!

**Comment by bluebill1049** [comment]
where did you place your `data-testid`? it needs to be on the actual dom element rather than the controller or you can use `getTextPlaceHolder` as well.
